Paystack/UI/PSTCKAuthViewController.m

```diff
@@ -43,9 +43,9 @@ - (void)viewDidLoad {
     UIBarButtonItem *cancelButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemCancel target:self action:@selector(tappedCancelButton:)];
     self.navigationItem.leftBarButtonItem = cancelButton;
     
-    self.authenticationWebView = [[UIWebView alloc] init];
-    self.authenticationWebView.delegate = self;
-    self.authenticationWebView.scalesPageToFit = NO;
```

Comment:
Do we have an equivalent setting for this? the 3DS pages used to be quite tiny is why we added this.

Reply:
I looked up the docs and I realized that the `scalesPageToFit` property is set to NO by default. That's why I removed that line. https://developer.apple.com/documentation/uikit/uiwebview/1617950-scalespagetofit?language=objc

Reply:
This documentation is for UIWebView... Also, explicitly setting it not to fit will ensure that any systems that have overridden the default will not have an issue

Reply:
Okay, got it. Figured out how to produce the same results with WKWebView is to use Javascript to modify the content width. It would be nice to test with the actual 3DS pages that are "tiny".

Reply:
You can try a live transaction with a zenith or gtb card, their 3ds windows were among those reported as showing up tiny then